owncloudData/src/main/java/com/owncloud/android/data/files/repository/OCFileRepository.kt

```diff
@@ -168,7 +168,7 @@ class OCFileRepository(
         val personalSpace = localSpacesDataSource.getPersonalSpaceForAccount(owner)
         if (personalSpace == null) {
             val legacyRootFolder = localFileDataSource.getFileByRemotePath(remotePath = ROOT_PATH, owner = owner, spaceId = null)
-            return legacyRootFolder!!
+            return legacyRootFolder ?: throw IllegalStateException("LegacyRootFolder not found")
```

Comment:
But is this exception caught anywhere? Because if not, we have the same problem (the app triggers an `IllegalStateException` instead of a `NullPointerException` but is crashing anyway)

Reply:
I have added a try catch.